src/views/pages/tasks/tasks-page.js

```diff
@@ -73,21 +72,38 @@ export class TasksPage extends Component {
       this.setState({
         selectedTask: this.props.tasks.first()
       })
-    }
+    }    
 
+    // prepare filter if exists
+    let curTasks = this.props.tasks;
```

Comment:
I love to have more descriptive names = such as `currentTasks` - it's just easier for me to understand